.github/workflows/ci.yml

```
@@ -5,6 +5,34 @@ on:
   pull_request:
 
 jobs:
+  build-openbsd:
+    runs-on: ubuntu-latest
+    defaults:
```

98414e7d28672aea7a218c691d34295d8934ae09:

For consistency with the naming of other jobs:: 

```suggestion
    runs-on: ubuntu-latest
    name: build • openbsd
    defaults:
```